org.telegram.desktop.yml

```diff
@@ -346,26 +326,34 @@ modules:
           versions:
             <: 2.73.0
 
-  - name: mozjpeg
-    buildsystem: cmake-ninja
-    builddir: true
-    config-opts:
-      - -DCMAKE_BUILD_TYPE=None
-      - -DCMAKE_INSTALL_DEFAULT_LIBDIR=lib
-      - -DENABLE_STATIC:BOOL=OFF
+  - name: jemalloc
```

Comment:
We prefer bundled sources, we have 3 official builds and we want to update only in one place whenever possible

Reply:
ok, restored, not sure why it failed without it.
Maybe it only failed using org.kde.Sdk or it was related with llvm missing...